Automattic/wp-calypso
# Plan: Add description to SFTP feature and not show empty tooltip

#### Proposed Changes

* Add description for SFTP feature.
* Do not show popover icon if feature description is unavailable.

#### Testing Instructions



* Go to` /plans/siteSlug` .
* Find feature `SFTP, SSH, WP-CLI, and Database access`.
* Click popover icon to see description,.

* Remove description from `client/lib/plans/features-list.js`.
* Verify popover icon is not shown for that feature.


**Before**
<img width="509" alt="Screenshot 2022-11-16 at 5 55 10 PM" src="https://user-images.githubusercontent.com/1182160/202244571-fcf2d168-599d-4b26-b0fa-ed2a423ce093.png">

**After**
![image](https://user-images.githubusercontent.com/47489215/202276152-1082690a-798f-42dd-a39b-858323fbeb2e.png)



#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/70078
